example/cumulus-tf/thin_egress_app.tf

```diff
@@ -1,79 +1,81 @@
-locals {
```

Comment:
instead of commenting everything out, can we just rename this file to thin_egress_app.tf.example?

Reply:
Oh yeah, that's a great idea! See the thread above re: conditionals vs. deploying both but this seems like a solid third option.